source/custom/nidcpower_service.custom.cpp

```diff
@@ -3,6 +3,8 @@
 #include <stdexcept>
 
 namespace nidcpower_grpc {
+using nidevice_grpc::converters::convert_from_grpc;
+using nidevice_grpc::converters::convert_to_grpc;
```

Comment:
Consider: There are existing uses of `nidevice_grpc::converters::convert_to_grpc::` in this file you could clean up if you don't mind.

Reply:
Done